src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,23 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){} // violation
```

Comment:
please make it `// 2 violations , '{' is not followed and preceded by whitespace.`

Reply:
Done @romani